Makefile

```diff
@@ -65,16 +65,19 @@ $(OUTPUT_DIRNAME)/$(DOC_FILENAME).pdf: $(DOC_FILES) $(FIGURE_FILES)
 else
 $(OUTPUT_DIRNAME)/$(DOC_FILENAME).pdf: $(DOC_FILES) $(FIGURE_FILES)
 	@mkdir -p $(OUTPUT_DIRNAME)/ && \
-	$(PANDOC) -f markdown_github -t latex -o $(PANDOC_DST)$@ $(patsubst %,$(PANDOC_SRC)%,$(DOC_FILES))
+	$(PANDOC) -f markdown_github -t latex -o $(PANDOC_DST)$@ $(patsubst %,$(PANDOC_SRC)%,$(DOC_FILES)) && \
```

Comment:
> i don't recall that to be the case. It's not like there is an implied `set -e` inside the target

    $ cat Makefile
    foo:
            false
            echo "still going?"
    $ make
    false
    Makefile:2: recipe for target 'foo' failed
    make: *** [foo] Error 1

If that's not convincing enough, I can try to hunt down some docs for you, just let me know.